src/components/RepoSearchForm.tsx

```diff
@@ -30,14 +42,26 @@ export const GithubSearchRepositories: React.FC = () => {
                             size="large"
                             value={searchName}
                             loading={isLoading}
-                            onSearch={onClick}
+                            onSearch={onSearch}
                             onChange={onChangeRepositoryTitle}
                     />
                 </Form>
-                <div className="itemsContainer">
-                    {list.map((item: GithubRepositoryItem) => <RepositoriesItems key={item.id} item={item} />)}
-                </div>
-                {/*<Pagination defaultCurrent={1} total={50} />*/}
+                <Spin className="loadingSpin" spinning={isLoading} size="large" tip="Loading...">
+                    <div className="itemsContainer">
+                        {list.map((item: GithubRepositoryItem) => <RepositoriesItems key={item.id} item={item} />)}
+                    </div>
+                </Spin>
+                {
+                        !!list.length &&
```

Comment:
No need for `!!`, the expression before `&&` would be casted to `boolean` type.

Reply:
Perhaps this is a bug, but without !! I have a string value of 0 displayed on a page.
That's why I used !! to convert to boolean list.length